sphinxcontrib/openapi.py

```diff
@@ -143,6 +291,11 @@ def _normalize_spec(spec, **options):
 
 def openapi2httpdomain(spec, **options):
```

Comment:
This class should probably be renamed, now that it's handling both OpenAPI 2.0 and OpenAPI 3.0.0.